app/Event.php

```diff
@@ -7,4 +7,8 @@
 class Event extends Model
 {
     //
+    function event_schedules()
+    {
+        return $this->hasMany('App\Event');
```

Comment:
ここは紐づけたいクラスが入るので、下記の記述が正しいです！
```suggestion
        return $this->hasMany('App\EventSchedule');
```

Reply:
で、↑はまず1対多のリレーションだったので、これを中間テーブル飛ばしてその先のテーブル（今回で言うとevent_schedules を飛ばしてその先の schedules テーブル ）の情報を取得できるようにするには

```
function schedules()
{
        return $this->belongsToMany('App\Schedule', 'event_schedules');
}
```

というような書き方をします！
（belongsToManyメソッドは初めてですね！）